src/Models/Model.php

```diff
@@ -109,6 +109,10 @@ protected static function bootTaggableTrait()
         static::$tagsModel = Tag::class;
     }
 
+    public function implementsTrait(string $trait): bool  {
```

Comment:
Could this be used instead? https://github.com/area17/twill/blob/2.x/src/Helpers/helpers.php#L49

Reply:
The class uses recursive is a laravel function that does the same.

https://laravel.com/docs/8.x/helpers#method-class-uses-recursive

But this is no longer needed as my change were not required :D